closed/make/modules/openjceplus/Copy.gmk

```diff
@@ -21,6 +21,15 @@
 include $(TOPDIR)/closed/CopySupport.gmk
 
 ifeq (true,$(BUILD_OPENJCEPLUS))
+  # Copy OpenJCEPlus legal files.
+  $(call openj9_copy_files,, \
+      $(OPENJCEPLUS_TOPDIR)/LICENSE \
+      $(LEGAL_DST_DIR)/OPENJCEPLUS_LICENSE)
+
+  $(call openj9_copy_files,, \
+      $(OPENJCEPLUS_TOPDIR)/NOTICES.md \
```

Comment:
This file does not appear to exist in https://github.com/IBM/OpenJCEPlus; that will need to change before this is safe to merge.

Reply:
If `NOTICES.md` won't be in the root repository (https://github.com/IBM/OpenJCEPlus), then this should be conditional on the existence of that file, which, based on https://github.com/ibmruntimes/OpenJCEPlus/pull/3, is only expected in that fork.

Reply:
This depends on https://github.com/ibmruntimes/OpenJCEPlus/pull/3, which should be merged first.